sass/forms/_buttons.scss

```diff
@@ -39,6 +39,7 @@ input[type="submit"] {
 	&.button-primary {
 		color: $color_white;
 		border: 1px solid $color_white;
+		margin-top: 25px;
```

Comment:
Looks like in general you're using tabs not spaces, I always recommend spaces :)